.github/workflows/docs.yml

```diff
@@ -37,21 +37,22 @@ jobs:
       with:
         hugo-version: '0.134.2'
 
-    - name: Build theme
+    - name: Get theme
       working-directory: ./docs/themes/geekdoc
-      run: |
-        npm install
-        npm run build
+      run: curl -L https://github.com/thegeeklab/hugo-geekdoc/releases/latest/download/hugo-geekdoc.tar.gz | tar -xz --strip-components=1
```

Comment:
This is more of a suggestion. Per the geekdoc docs, since you are just using the standard theme, I think you can save most of the time in this step by just using the pre-built tarball here.

If you like it (and it works) I can remove the submodule for geekdocs too.

Reply:
> [publish-doc](https://github.com/emacs-eask/cli/actions/runs/14000551603/job/39205778866?pr=310#logs)
> succeeded 8 minutes ago in 10s

Oh yeah!

Reply:
I checked it locally by:
* running the `curl` command in `docs/themes/geekdocs`
* starting hugo using a container (run in the `docs` directory): `docker run --network=host --mount type=bind,source=.,destination=/mine/ -w /mine peaceiris/hugo:v0.134.2 server`
* visit the ip address it suggests

Looks like the theme is loaded correctly.

Reply:
Thank you for the suggestion! I appreciate it, but I prefer using the old method as it’s more reproducible for me. 😅

Reply:
> I checked it locally

BTW this was just my way of avoiding installing hugo, and guaranteeing the versions matched upstream.

For local development, you could absolutely keep the submodule method. It would match this one unless you switched to a different branch on the `hugo-geekdoc` repo.

Could also put the `curl` command into a Make rule that matches what happens in this file, so (assuming you have hugo locally), you just do `make docs`: https://geekdocs.de/usage/getting-started/#use-a-makefile

Reply:
> BTW this was just my way of avoiding installing hugo, and guaranteeing the versions matched upstream.
> 
> For local development, you could absolutely keep the submodule method. It would match this one unless you switched to a different branch on the hugo-geekdoc repo.

I see the YAML file as more of a reference for setting up the environment. I also want to eliminate other potential variables to improve the debugging experience. While this approach may increase CI runtime, it’s a trade-off that will save me time in the long run. 🤔 

I guess I'm being a bit selfish here... but I might change my mind in the future. 😉

Reply:
No worries. Can always refer back to this! Will drop this commit